string frame_id
mines/region[] region_trajectory
string state
string[] task_trajectory
int8 task_index

